### PR TITLE
Feature: blog cms detail page layout

### PR DESCRIPTION
**Pull request by viennt**

### CODE REVIEW

File: src/Resources/app/administration/src/module/blog-module/extension/component/cms/sas-cms-sidebar/sas-cms-sidebar.html.twig [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+            slot="content"
+            columns="100%"
+            gap="20px"
+            class="sw-cms-sidebar__settings"
+        >
+
+            <div>
+                {% block sas_blog_cms_sidebar_page_settings_visibility_field %}
+                <sw-switch-field
+                    style="margin-top:0;margin-bottom:0"
```
Comment (crisalder2806) [inline src/Resources/app/administration/src/module/blog-module/extension/component/cms/sas-cms-sidebar/sas-cms-sidebar.html.twig:101]: Use a css class whenever possible

File: src/Resources/app/administration/src/module/blog-module/extension/component/cms/sas-cms-sidebar/sas-cms-sidebar.html.twig [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+
+    {% block sas_blog_cms_sidebar_page_serp %}
+    <sw-sidebar-collapse :expand-on-loading="true">
+
+        {% block sas_blog_cms_sidebar_page_serp_header %}
+            <span slot="header">{{ $tc('sas-blog.detail.serp.title') }}</span>
+        {% endblock %}
+
+        {% block sas_blog_cms_sidebar_page_serp_form %}
+            <sw-container
```
Comment (crisalder2806) [inline src/Resources/app/administration/src/module/blog-module/extension/component/cms/sas-cms-sidebar/sas-cms-sidebar.html.twig:174]: Check the indent here.

File: src/Resources/app/administration/src/module/blog-module/page/sas-blog-detail/sas-blog-detail.html.twig [hunk 3]
```diff
@@ -1,224 +1,105 @@
...
+<div class="sw-cms-detail__empty-stage-content">
+    <h2 class="sw-cms-detail__empty-stage-headline">
+        {{ $tc('sw-cms.detail.label.headlineEmptyState') }}
+    </h2>
+    <p class="sw-cms-detail__empty-stage-claim">
+        {{ $tc('sw-cms.detail.label.claimEmptyState') }}
+    </p>
+    <sw-cms-stage-add-section
+        :key="0"
+        ref="cmsStageAddSection"
```
Comment (crisalder2806) [inline src/Resources/app/administration/src/module/blog-module/page/sas-blog-detail/sas-blog-detail.html.twig:54]: Put ref before `:key`

File: src/Resources/app/administration/src/module/blog-module/extension/component/cms/sas-cms-sidebar/index.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+            this.blog.mediaId = null;
+            this.blog.media = null;
+        },
+
+        onMediaDropped(dropItem) {
+            this.onSetMediaItem({ targetId: dropItem.id });
+        },
+
+        openMediaSidebar() {
+            this.$parent.$parent.$parent.$parent.$refs.mediaSidebarItem.openContent();
```
Comment (crisalder2806) [inline src/Resources/app/administration/src/module/blog-module/extension/component/cms/sas-cms-sidebar/index.js:81]: I don't see this being used anywhere in the code. Can you check it again?

File: src/Resources/app/administration/src/module/blog-module/extension/component/cms/sas-cms-sidebar/sas-cms-sidebar.html.twig [hunk 5]
```diff
@@ -0,0 +1,213 @@
+{% block sw_cms_sidebar_page_settings %}
+<sw-sidebar-item
+    ref="blogDetailSidebar"
+    icon="default-badge-info"
+    :title="$tc('sas-blog.detail.sidebar.heading')"
+    :has-simple-badge="blogModuleDetailError"
+    badge-type="error"
```
Comment (crisalder2806) [inline src/Resources/app/administration/src/module/blog-module/extension/component/cms/sas-cms-sidebar/sas-cms-sidebar.html.twig:7]: Put `badge-type` before `:title`

File: src/Resources/app/administration/src/module/blog-module/extension/component/cms/sas-cms-sidebar/sas-cms-sidebar.html.twig [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+                :error="blogTitleError">
+            </sas-text-field>
+            {% endblock %}
+
+            {% block sas_blog_cms_sidebar_page_basic_information_slug_field %}
+            <sw-text-field
+                required
+                disabled
+                v-model="blog.slug"
+                placeholder="Undefined"
```
Comment (crisalder2806) [inline src/Resources/app/administration/src/module/blog-module/extension/component/cms/sas-cms-sidebar/sas-cms-sidebar.html.twig:41]: Put this in the snippets.

File: src/Resources/app/administration/src/module/blog-module/extension/component/cms/sas-cms-sidebar/index.js (hunk 4 above)
Comment (viennt) [reply src/Resources/app/administration/src/module/blog-module/extension/component/cms/sas-cms-sidebar/index.js:81]: In the master branch, it's also a redundant method as well. I'll remove it then.

Comment (ChristopherDosin) [reply src/Resources/app/administration/src/module/blog-module/extension/component/cms/sas-cms-sidebar/index.js:81]: I think this was dirty code from the past. If possible clean it up.

**Review by crisalder2806 — APPROVED** [review]

**Pull request merged**